### ISSUE TITLE
ImportError: No module named _pluggy when updating to pytest 3.3.0

### ISSUE THREAD

**Issue by stephendonner** [davehunt/pytest-metadata#5]
Looks like if one updates to pytest 3.3.0, you get an import error from pytest-metadata, as follows (due to changes from https://github.com/pytest-dev/pytest/pull/2719):

```
Started by user sdonner@mozilla.com
Checking out git ${REPOSITORY} into /var/lib/jenkins/jobs/fxapom.adhoc/workspace@script to read Jenkinsfile
Wiping out workspace first.
Cloning the remote Git repository
Cloning repository https://github.com/stephendonner/fxapom.git
 > git init /var/lib/jenkins/jobs/fxapom.adhoc/workspace@script # timeout=10
Fetching upstream changes from https://github.com/stephendonner/fxapom.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/stephendonner/fxapom.git +refs/heads/*:refs/remotes/origin/*
 > git config remote.origin.url https://github.com/stephendonner/fxapom.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/stephendonner/fxapom.git # timeout=10
Fetching upstream changes from https://github.com/stephendonner/fxapom.git
 > git fetch --tags --progress https://github.com/stephendonner/fxapom.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse origin/pytest-3.3.0^{commit} # timeout=10
Checking out Revision e4b502321ab03406c602b744a3af35ed7cebcc29 (origin/pytest-3.3.0)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f e4b502321ab03406c602b744a3af35ed7cebcc29
Commit message: "Update pytest to 3.3.0"
First time build. Skipping changelog.
[Pipeline] library
Loading library fxtest@1.9
 > git rev-parse --is-inside-work-tree # timeout=10
Setting origin to https://github.com/mozilla/fxtest-jenkins-pipeline.git
 > git config remote.origin.url https://github.com/mozilla/fxtest-jenkins-pipeline.git # timeout=10
Fetching origin...
Fetching upstream changes from origin
 > git --version # timeout=10
 > git fetch --tags --progress origin +refs/heads/*:refs/remotes/origin/*
 > git rev-parse 1.9^{commit} # timeout=10
Wiping out workspace first.
Cloning the remote Git repository
Cloning with configured refspecs honoured and without tags
Cloning repository https://github.com/mozilla/fxtest-jenkins-pipeline.git
 > git init /var/lib/jenkins/jobs/fxapom.adhoc/workspace@libs/fxtest # timeout=10
Fetching upstream changes from https://github.com/mozilla/fxtest-jenkins-pipeline.git
 > git --version # timeout=10
 > git fetch --no-tags --progress https://github.com/mozilla/fxtest-jenkins-pipeline.git +refs/heads/*:refs/remotes/origin/*
 > git config remote.origin.url https://github.com/mozilla/fxtest-jenkins-pipeline.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/mozilla/fxtest-jenkins-pipeline.git # timeout=10
Fetching without tags
Fetching upstream changes from https://github.com/mozilla/fxtest-jenkins-pipeline.git
 > git fetch --no-tags --progress https://github.com/mozilla/fxtest-jenkins-pipeline.git +refs/heads/*:refs/remotes/origin/*
Checking out Revision 3d5e2e860c51a1572c84e952c59bf4a035a5443c (1.9)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 3d5e2e860c51a1572c84e952c59bf4a035a5443c
Commit message: "Merge pull request #35 from stephendonner/fix-API-URL"
 > git rev-list 3d5e2e860c51a1572c84e952c59bf4a035a5443c # timeout=10
[Pipeline] node
Running on centos7-1-qatest-slave.fxtestpreprod.jenkins.stage.mozaws.net in /home/jenkins/workspace/fxapom.adhoc
[Pipeline] {
[Pipeline] stage
[Pipeline] { (Declarative: Checkout SCM)
[Pipeline] checkout
Wiping out workspace first.
Cloning the remote Git repository
Cloning repository https://github.com/stephendonner/fxapom.git
 > git init /home/jenkins/workspace/fxapom.adhoc # timeout=10
Fetching upstream changes from https://github.com/stephendonner/fxapom.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/stephendonner/fxapom.git +refs/heads/*:refs/remotes/origin/*
 > git config remote.origin.url https://github.com/stephendonner/fxapom.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/stephendonner/fxapom.git # timeout=10
Fetching upstream changes from https://github.com/stephendonner/fxapom.git
 > git fetch --tags --progress https://github.com/stephendonner/fxapom.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse origin/pytest-3.3.0^{commit} # timeout=10
Checking out Revision e4b502321ab03406c602b744a3af35ed7cebcc29 (origin/pytest-3.3.0)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f e4b502321ab03406c602b744a3af35ed7cebcc29
Commit message: "Update pytest to 3.3.0"
[Pipeline] }
[Pipeline] // stage
[Pipeline] withEnv
[Pipeline] {
[Pipeline] withCredentials
[Pipeline] {
[Pipeline] withEnv
[Pipeline] {
[Pipeline] ansiColor
[Pipeline] {
[Pipeline] timestamps
[Pipeline] {
[Pipeline] timeout
Timeout set to expire in 1 hr 0 min
[Pipeline] {
[Pipeline] stage
[Pipeline] { (Lint)
[Pipeline] sh
[fxapom.adhoc] Running shell script
+ tox -e flake8
GLOB sdist-make: /home/jenkins/workspace/fxapom.adhoc/setup.py
flake8 create: /home/jenkins/workspace/fxapom.adhoc/.tox/flake8
flake8 installdeps: -rrequirements/flake8.txt
flake8 installed: configparser==3.5.0,enum34==1.1.6,flake8==3.5.0,flake8-isort==2.2.2,flake8-polyfill==1.0.1,isort==4.2.15,mccabe==0.6.1,pycodestyle==2.3.1,pyflakes==1.6.0,testfixtures==5.3.1
flake8 runtests: PYTHONHASHSEED='4275573629'
flake8 runtests: commands[0] | flake8 .
___________________________________ summary ____________________________________
  flake8: commands succeeded
  congratulations :)
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Test)
[Pipeline] writeFile
[Pipeline] sh
[fxapom.adhoc] Running shell script
+ tox -e py27
GLOB sdist-make: /home/jenkins/workspace/fxapom.adhoc/setup.py
py27 create: /home/jenkins/workspace/fxapom.adhoc/.tox/py27
py27 installdeps: -rrequirements/py27-tests.txt
py27 inst: /home/jenkins/workspace/fxapom.adhoc/.tox/dist/fxapom-1.9.1.dev110-ge4b5023.zip
Post stage
py27 installed: apipkg==1.4,asn1crypto==0.23.0,attrs==17.3.0,blessings==1.6,certifi==2017.11.5,cffi==1.11.2,chardet==3.0.4,cryptography==2.1.3,enum34==1.1.6,execnet==1.5.0,funcsigs==1.0.2,fxapom===1.9.1.dev110-ge4b5023,hawkauthlib==2.0.0,idna==2.6,ipaddress==1.0.18,mock==2.0.0,mozlog==3.5,pbr==3.1.1,pluggy==0.6.0,py==1.5.2,PyBrowserID==0.12.0,pycparser==2.18,PyFxA==0.3.0,PyPOM==1.2.0,pytest==3.3.0,pytest-base-url==1.4.1,pytest-forked==0.2,pytest-html==1.16.0,pytest-metadata==1.5.0,pytest-selenium==1.11.2,pytest-variables==1.7.0,pytest-xdist==1.20.1,requests==2.18.4,selenium==3.7.0,six==1.11.0,urllib3==1.22,WebOb==1.7.4,zope.component==4.4.1,zope.event==4.3.0,zope.interface==4.4.3
py27 runtests: PYTHONHASHSEED='387830163'
py27 runtests: commands[0] | pytest --junit-xml=results/py27.xml --html=results/py27.html --self-contained-html tests
Traceback (most recent call last):
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/bin/pytest", line 11, in <module>
    sys.exit(main())
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/_pytest/config.py", line 50, in main
    config = _prepareconfig(args, plugins)
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/_pytest/config.py", line 160, in _prepareconfig
    pluginmanager=pluginmanager, args=args)
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/pluggy/__init__.py", line 617, in __call__
    return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/pluggy/__init__.py", line 222, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/pluggy/__init__.py", line 216, in <lambda>
    firstresult=hook.spec_opts.get('firstresult'),
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/pluggy/callers.py", line 196, in _multicall
    gen.send(outcome)
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/_pytest/helpconfig.py", line 68, in pytest_cmdline_parse
    config = outcome.get_result()
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/pluggy/callers.py", line 77, in get_result
    _reraise(*ex)  # noqa
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/pluggy/callers.py", line 180, in _multicall
    res = hook_impl.function(*args)
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/_pytest/config.py", line 942, in pytest_cmdline_parse
    self.parse(args)
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/_pytest/config.py", line 1107, in parse
    self._preparse(args, addopts=addopts)
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/_pytest/config.py", line 1070, in _preparse
    self.pluginmanager.load_setuptools_entrypoints('pytest11')
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/pluggy/__init__.py", line 397, in load_setuptools_entrypoints
    plugin = ep.load()
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2405, in load
    return self.resolve()
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2411, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/_pytest/assertion/rewrite.py", line 212, in load_module
    py.builtin.exec_(co, mod.__dict__)
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/py/_builtin.py", line 221, in exec_
    exec2(obj, globals, locals)
  File "<string>", line 7, in exec2
  File "/home/jenkins/workspace/fxapom.adhoc/.tox/py27/lib/python2.7/site-packages/pytest_metadata/plugin.py", line 8, in <module>
    import _pytest._pluggy as pluggy
ImportError: No module named _pluggy
ERROR: InvocationError: '/home/jenkins/workspace/fxapom.adhoc/.tox/py27/bin/pytest --junit-xml=results/py27.xml --html=results/py27.html --self-contained-html tests'
___________________________________ summary ____________________________________
ERROR:   py27: commands failed
[Pipeline] archiveArtifacts
Archiving artifacts
[Pipeline] junit
Recording test results
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Declarative: Post Actions)
[Pipeline] sh
[fxapom.adhoc] Running shell script
+ echo NICK fxtest-68
+ openssl s_client -connect irc.mozilla.org:6697
+ echo USER fxtest-68 8 capabilities.json CHANGES.rst fxapom fxapom.egg-info Jenkinsfile LICENSE README.rst requirements setup.cfg setup.py setuptools_scm-1.15.6-py2.7.egg tests tox.ini : fxtest-68
+ sleep 5
depth=2 C = US, O = DigiCert Inc, OU = www.digicert.com, CN = DigiCert Global Root CA
verify return:1
depth=1 C = US, O = DigiCert Inc, CN = DigiCert SHA2 Secure Server CA
verify return:1
depth=0 C = US, ST = California, L = Mountain View, O = Mozilla Foundation, OU = IT, CN = irc.mozilla.org
verify return:1
CONNECTED(00000003)
---
Certificate chain
 0 s:/C=US/ST=California/L=Mountain View/O=Mozilla Foundation/OU=IT/CN=irc.mozilla.org
   i:/C=US/O=DigiCert Inc/CN=DigiCert SHA2 Secure Server CA
 1 s:/C=US/O=DigiCert Inc/CN=DigiCert SHA2 Secure Server CA
   i:/C=US/O=DigiCert Inc/OU=www.digicert.com/CN=DigiCert Global Root CA
---
Server certificate
-----BEGIN CERTIFICATE-----
MIIFwDCCBKigAwIBAgIQD77v0VstmzNr7TRq4t56AzANBgkqhkiG9w0BAQsFADBN
MQswCQYDVQQGEwJVUzEVMBMGA1UEChMMRGlnaUNlcnQgSW5jMScwJQYDVQQDEx5E
aWdpQ2VydCBTSEEyIFNlY3VyZSBTZXJ2ZXIgQ0EwHhcNMTcwODA5MDAwMDAwWhcN
MjAwODEzMTIwMDAwWjB+MQswCQYDVQQGEwJVUzETMBEGA1UECBMKQ2FsaWZvcm5p
YTEWMBQGA1UEBxMNTW91bnRhaW4gVmlldzEbMBkGA1UEChMSTW96aWxsYSBGb3Vu
ZGF0aW9uMQswCQYDVQQLEwJJVDEYMBYGA1UEAxMPaXJjLm1vemlsbGEub3JnMIIB
IjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAxIYfBDX0OreQgLT7MEHgaOg9
oDVT7GBYmUF27sTsdUYLzsHou//Frgg5ZovyLNzaDOQmADExNANMzna50jZJ5aXI
PBmqia3taq8ZSm8+4UaBiyC8gj1k+XdIEnpYCHgTxHX2gv/24+8Fc9KsppRsvNvq
ptRk6ziOyHUwBft7Ml+yLQfCd6YpsDzSajMyuouSsQQj8i5H5r9/bKozmDmsxNJh
p86wP8BOG3VWqc7h2rrQkkaFB1b80lrQe0Cm4e/VhPSLqukcZJinZ6tPpwbYhuql
yw75AaUfjyw1p2WzgVL085Sy3oCHriWgCzs8j0hk+dZ2HQSpc4SLsOj+7ZvZQwID
AQABo4ICaTCCAmUwHwYDVR0jBBgwFoAUD4BhHIIxYdUvKOeNRji0LOHG2eIwHQYD
VR0OBBYEFDn3LPIgvOVUN/h5bOAyLqI98f2RMIGsBgNVHREEgaQwgaGCD2lyYy5t
b3ppbGxhLm9yZ4ITYnJ1Zm9yZC5tb3ppbGxhLm9yZ4IRZnJpcHAubW96aWxsYS5v
cmeCEWxldmluLm1vemlsbGEub3JnghFiZWxldy5tb3ppbGxhLm9yZ4IUaXJjY2xv
dWQubW96aWxsYS5vcmeCFGlyY2Nsb3VkLm1vemlsbGEuY29tghRpcmNjbG91ZC5h
bGxpem9tLm9yZzAOBgNVHQ8BAf8EBAMCBaAwHQYDVR0lBBYwFAYIKwYBBQUHAwEG
CCsGAQUFBwMCMGsGA1UdHwRkMGIwL6AtoCuGKWh0dHA6Ly9jcmwzLmRpZ2ljZXJ0
LmNvbS9zc2NhLXNoYTItZzEuY3JsMC+gLaArhilodHRwOi8vY3JsNC5kaWdpY2Vy
dC5jb20vc3NjYS1zaGEyLWcxLmNybDBMBgNVHSAERTBDMDcGCWCGSAGG/WwBATAq
MCgGCCsGAQUFBwIBFhxodHRwczovL3d3dy5kaWdpY2VydC5jb20vQ1BTMAgGBmeB
DAECAjB8BggrBgEFBQcBAQRwMG4wJAYIKwYBBQUHMAGGGGh0dHA6Ly9vY3NwLmRp
Z2ljZXJ0LmNvbTBGBggrBgEFBQcwAoY6aHR0cDovL2NhY2VydHMuZGlnaWNlcnQu
Y29tL0RpZ2lDZXJ0U0hBMlNlY3VyZVNlcnZlckNBLmNydDAMBgNVHRMBAf8EAjAA
MA0GCSqGSIb3DQEBCwUAA4IBAQAMAn/IA6VRRqvmK5eZLnvyD27PfG/WcJ3XySYG
IdNGtVUGSSkzqoYr8HWdN0/WGrRsPN9y7SN+AtooaKULmdXeg72j9blQdz2CKfNd
sqUVHMPaB2JUPfJrUpHGfVMSiSm4xmINMJ3Kw6XE+F30MlyUIQzInDuoUlvnG2xX
PG1JSIMhtxcm1Vu2hVgndTT74HQsSY1ls4RVeF5IBJ0VlCIctAarFBJ56MPG2v8o
fN5QKQiDzT93J3oysMsg/KMwAvT7hwe7V0xtudIj3TYixv1smPLPfIMtFjrasjdJ
II1Qkg4KyjKPKGIRtRQWQHg7Jzxp8dMYstcJ3Q/gpRf/dmci
-----END CERTIFICATE-----
subject=/C=US/ST=California/L=Mountain View/O=Mozilla Foundation/OU=IT/CN=irc.mozilla.org
issuer=/C=US/O=DigiCert Inc/CN=DigiCert SHA2 Secure Server CA
---
No client certificate CA names sent
Server Temp Key: DH, 1024 bits
---
SSL handshake has read 3446 bytes and written 489 bytes
---
New, TLSv1/SSLv3, Cipher is DHE-RSA-AES256-SHA256
Server public key is 2048 bit
Secure Renegotiation IS supported
Compression: NONE
Expansion: NONE
SSL-Session:
    Protocol  : TLSv1.2
    Cipher    : DHE-RSA-AES256-SHA256
    Session-ID: 90A66C32C9BE161B615C4DFB30F4E8F3841446722946930E0B5CBAB696FB52BC
    Session-ID-ctx: 
    Master-Key: C24BA7D858765AB40A09CF3F2E8E4447E2565518283109AD673DDEEEDFA486BFCF4A00B4AA6AE42F99E983BA2B5ED717
    Key-Arg   : None
    Krb5 Principal: None
    PSK identity: None
    PSK identity hint: None
    Start Time: 1511825928
    Timeout   : 300 (sec)
    Verify return code: 0 (ok)
---
:belew.mozilla.org NOTICE Auth :*** Looking up your hostname...
:belew.mozilla.org NOTICE Auth :*** Found your hostname (ec2-54-88-37-135.compute-1.amazonaws.com)
:belew.mozilla.org NOTICE Auth :Welcome to Mozilla!
:belew.mozilla.org 001 fxtest-68 :Welcome to the Mozilla IRC Network fxtest-68!fxtest-68@ec2-54-88-37-135.compute-1.amazonaws.com
:belew.mozilla.org 002 fxtest-68 :Your host is belew.mozilla.org, running version InspIRCd-2.0
:belew.mozilla.org 003 fxtest-68 :This server was created 11:39:26 Sep  3 2016
:belew.mozilla.org 004 fxtest-68 belew.mozilla.org InspIRCd-2.0 BHIRSWacdhikorswx ABCDFKLMNOQRSTYZabcefghijklmnopqrstuvz FLYZabefghjkloqv
:belew.mozilla.org 005 fxtest-68 AWAYLEN=200 CASEMAPPING=rfc1459 CHANMODES=Zbeg,k,FLfjl,ABCDKMNOQRSTcimnprstuz CHANNELLEN=64 CHANTYPES=# CHARSET=ascii ELIST=MU ESILENCE EXCEPTS=e EXTBAN=,ABCNOQRSTUcmprz FNC KICKLEN=255 MAP :are supported by this server
:belew.mozilla.org 005 fxtest-68 MAXBANS=60 MAXCHANNELS=100 MAXPARA=32 MAXTARGETS=20 MODES=20 NAMESX NETWORK=Mozilla NICKLEN=31 OPERLOG OVERRIDE PREFIX=(Yqaohv)!~&@%+ SECURELIST SILENCE=32 :are supported by this server
:belew.mozilla.org 005 fxtest-68 SSL=[::]:6697 STATUSMSG=!~&@%+ TOPICLEN=307 UHNAMES USERIP VBANLIST WALLCHOPS WALLVOICES WATCH=32 :are supported by this server
:belew.mozilla.org 042 fxtest-68 203BCLCT4 :your unique ID
:belew.mozilla.org 375 fxtest-68 :belew.mozilla.org message of the day
:belew.mozilla.org 372 fxtest-68 :-                      __         __
:belew.mozilla.org 372 fxtest-68 :-                     / /_  ___  / /__ _      __
:belew.mozilla.org 372 fxtest-68 :-                    / __ \/ _ \/ / _ \ | /| / /
:belew.mozilla.org 372 fxtest-68 :-                   / /_/ /  __/ /  __/ |/ |/ /
:belew.mozilla.org 372 fxtest-68 :-                  /_.___/\___/_/\___/|__/|__/
:belew.mozilla.org 372 fxtest-68 :- 
:belew.mozilla.org 372 fxtest-68 :- 
:belew.mozilla.org 372 fxtest-68 :-                  M O Z I L L A   D O T   O R G
:belew.mozilla.org 372 fxtest-68 :-                    https://www.mozilla.org/
:belew.mozilla.org 372 fxtest-68 :- 
:belew.mozilla.org 372 fxtest-68 :- =================================================================
:belew.mozilla.org 372 fxtest-68 :-  This server is for use by Mozilla developers, testers, and
:belew.mozilla.org 372 fxtest-68 :-  users. If you are not one, please find another server.
:belew.mozilla.org 372 fxtest-68 :- 
:belew.mozilla.org 372 fxtest-68 :-  NOTICE: Your IP address will be portscanned by our open
:belew.mozilla.org 372 fxtest-68 :-  proxy tester.  See https://wiki.mozilla.org/IRC for details.
:belew.mozilla.org 372 fxtest-68 :- 
:belew.mozilla.org 372 fxtest-68 :- =================================================================
:belew.mozilla.org 372 fxtest-68 :-  Project information:
:belew.mozilla.org 372 fxtest-68 :-   Firefox                https://support.mozilla.org/
:belew.mozilla.org 372 fxtest-68 :-   Mobile                 https://support.mozilla.org/mobile
:belew.mozilla.org 372 fxtest-68 :-   Thunderbird            https://support.mozilla.org/products/thunderbird
:belew.mozilla.org 372 fxtest-68 :-   Boot2Gecko (FirefoxOS) https://support.mozilla.org/products/firefox-os
:belew.mozilla.org 372 fxtest-68 :-   Webmaker               https://support.mozilla.org/products/webmaker
:belew.mozilla.org 372 fxtest-68 :-  
:belew.mozilla.org 372 fxtest-68 :-  Help for new contributors:
:belew.mozilla.org 372 fxtest-68 :-   https://developer.mozilla.org/Introduction
:belew.mozilla.org 372 fxtest-68 :-  
:belew.mozilla.org 372 fxtest-68 :- =================================================================
:belew.mozilla.org 372 fxtest-68 :-  Ports:
:belew.mozilla.org 372 fxtest-68 :-   Non-SSL: 6665 - 6669
:belew.mozilla.org 372 fxtest-68 :-       SSL: 6697, 8443
:belew.mozilla.org 372 fxtest-68 :- =================================================================
:belew.mozilla.org 372 fxtest-68 :- 
:belew.mozilla.org 372 fxtest-68 :-  For a list of other helpful channels please visit
:belew.mozilla.org 372 fxtest-68 :-  https://wiki.mozilla.org/IRC or use the /list command.
:belew.mozilla.org 372 fxtest-68 :-  Additional help with IRC at http://www.irchelp.org/
:belew.mozilla.org 372 fxtest-68 :- 
:belew.mozilla.org 376 fxtest-68 :End of message of the day.
:belew.mozilla.org 251 fxtest-68 :There are 5125 users and 788 invisible on 4 servers
:belew.mozilla.org 252 fxtest-68 15 :operator(s) online
:belew.mozilla.org 253 fxtest-68 17 :unknown connections
:belew.mozilla.org 254 fxtest-68 1988 :channels formed
:belew.mozilla.org 255 fxtest-68 :I have 2062 clients and 2 servers
:belew.mozilla.org 265 fxtest-68 :Current Local Users: 2062  Max: 3052
:belew.mozilla.org 266 fxtest-68 :Current Global Users: 5913  Max: 6188
:belew.mozilla.org 396 fxtest-68 moz-hhvi8r.compute-1.amazonaws.com :is now your displayed host
:fxtest-68!fxtest-68@moz-hhvi8r.compute-1.amazonaws.com MODE fxtest-68 +x
:belew.mozilla.org NOTICE fxtest-68 :*** You are connected using SSL cipher "DHE-RSA-AES-256-CBC-SHA256"
+ echo 'JOIN #fx-test-alerts'
+ echo 'NOTICE #fx-test-alerts :Project fxapom.adhoc build #68: FAILURE: https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/fxapom.adhoc/68/'
+ echo QUIT
DONE
[Pipeline] emailext
Sending email to: fte-ci@mozilla.com
[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // timeout
[Pipeline] }
[Pipeline] // timestamps
[Pipeline] }
[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // withEnv
[Pipeline] }
[Pipeline] // withCredentials
[Pipeline] }
[Pipeline] // withEnv
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
ERROR: script returned exit code 1
Finished: FAILURE
```

**Comment by davehunt** [comment]
Thanks for raising this @stephendonner! I've fixed it and released [1.5.1](https://pypi.python.org/pypi/pytest-metadata/1.5.1).

**Comment by benjaminrigaud** [comment]
thank you @davehunt ! you may want to pin it in https://github.com/pytest-dev/pytest-html/blob/master/setup.py
